package.json

```diff
@@ -58,14 +59,7 @@
   "homepage": "https://github.com/trussworks/react-uswds#readme",
   "peerDependencies": {
     "react": "^16.x",
-    "react-dom": "^16.x"
-  },
-  "dependencies": {
```

Comment:
I'm trying to be better versed on the different dependency types here, looking at https://classic.yarnpkg.com/en/docs/dependency-types/

> #### Dependencies
> These are your normal dependencies, or rather ones that you need when running your code (e.g. React or ImmutableJS).

> #### devDependencies
> These are your development dependencies. Dependencies that you need at some point in the development workflow but not while running your code (e.g. Babel or Flow).

> #### peerDependencies
> Peer dependencies are a special type of dependency that would only ever come up if you were publishing your own package.
> Having a peer dependency means that your package needs a dependency that is the same exact dependency as the person installing your package. This is useful for packages like react that need to have a single copy of react-dom that is also used by the person installing it.

Is the reasoning that things like `react` and `uswds` only need to be `peerDependencies` that react-uswds itself doesn't "run", so we expect only the implementing App to include it?

---

Second question, mostly a point of curiosity: 

I've used some FE component libraries that include all their own styles transitively before. For react-uswds it looks like we require the implementing project to include USWDS styles on its own as well. Is this intentional because the assumed use-case is for our components to supplement a project already using USWDS in some capacity?

Reply:
@brandonlenz The answer to your question (in my understanding at least) is yes.